repl/src/main/resources/fake_shell.py

```diff
@@ -261,26 +268,32 @@ def magic_table_convert_map(m):
 magic_table_types = {
     type(None): lambda x: ('NULL_TYPE', x),
     bool: lambda x: ('BOOLEAN_TYPE', x),
-    int: lambda x: ('INT_TYPE', x),
-    long: lambda x: ('BIGINT_TYPE', x),
+    int: lambda x: ('INT_TYPE', x),    
     float: lambda x: ('DOUBLE_TYPE', x),
     str: lambda x: ('STRING_TYPE', str(x)),
-    unicode: lambda x: ('STRING_TYPE', x.encode('utf-8')),
     datetime.date: lambda x: ('DATE_TYPE', str(x)),
     datetime.datetime: lambda x: ('TIMESTAMP_TYPE', str(x)),
     decimal.Decimal: lambda x: ('DECIMAL_TYPE', str(x)),
     tuple: magic_table_convert_seq,
     list: magic_table_convert_seq,
-    dict: magic_table_convert_map,
+    dict: magic_table_convert_map,    
```

Comment:
This seems to be a meaningless change. Would you mind to strip ending spaces (there are some in other places in your PR)?